pom.xml

```diff
@@ -19,20 +19,36 @@
     <build>
         <directory>test-outputs/target</directory>
     </build>
+
     <dependencies>
         <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>${seleniumVersion}</version>
```

Comment:
Why did you delete the ${seleniumVersion}? I have created it and it binds to the variable on the properties tag for easy handling of the versioning.